src/tests/class_tests/openms/source/IdentificationData_test.cpp

```diff
@@ -436,7 +436,7 @@ END_SECTION
 
 START_SECTION((result<ScoreTypeRef, bool> findScoreType(const String& score_name, ProcessingSoftwareRef software_ref) const))
 {
-  Software sw("Other");
+  Software sw("Other","1.0");
```

Comment:
This shouldn't be necessary. What was your reason for making this change?

Reply:
Oh, it's probably that "result" in the function signature (after "START_SECTION") should be "pair". I'll fix it.

Reply:
Nope, I see the problem is that my changes to `Software` got lost. I changed the constructor to `explicit Software(const String& name = "", const String& version = "")` (and removed the annoying spaces before the `&` of references).
@poshul, can you please update "Software.h" and "Software.cpp" with the versions from my branch?

Reply:
PR 37 (https://github.com/poshul/OpenMS/pull/37) didn't compile due to a lack of constructor signature mismatch. I fixed that in the least invasive way possible. I will pull in the rest of your changes.